core/serialization.h

```diff
@@ -157,6 +157,13 @@ void serialization_utils_writeUint16(void *dest, const uint16_t src, uint32_t *c
 ///
 void serialization_utils_writeUint32(void *dest, const uint32_t src, uint32_t *cursor);
 
+// MARK: - Baked files -
+
+bool serialization_save_baked_file(const Shape *s,
+                                   uint32_t paletteHash,
```

Comment:
I see we use a shape hash here in some situation, coming from `shape_get_baked_lighting_hash`. 
The name could be more generic, (just "hash"?), and we could accept uint64_t not to lose precision. (casting `uint32_t` hashes into `uint64_t` ones)